cqlshlib/saferscanner.py

```diff
@@ -81,4 +85,24 @@ def __init__(self, lexicon, flags=0):
         self.scanner = re.sre_compile.compile(p)
 
 
-SaferScanner = Py38SaferScanner if version_info >= (3, 8) else Py36SaferScanner
+class Py311SaferScanner(SaferScannerBase):
+
+    def __init__(self, lexicon, flags=0):
+        self.lexicon = lexicon
+        p = []
+        s = re._parser.State()
+        s.flags = flags
+        for phrase, action in lexicon:
+            gid = s.opengroup()
+            p.append(re._parser.SubPattern(s, [(SUBPATTERN, (gid, 0, 0, re._parser.parse(phrase, flags))), ]))
+            s.closegroup(gid, p[-1])
+        p = re._parser.SubPattern(s, [(BRANCH, (None, p))])
+        self.p = p
+        self.scanner = re._compiler.compile(p)
+
+
+SaferScanner = Py36SaferScanner if six.PY3 else Py2SaferScanner
```

Comment:
Haha, I noticed this and went digging to see why we needed `six`, only to discover you're one step ahead of me!

https://issues.apache.org/jira/browse/CASSANDRA-18088?focusedCommentId=17703393&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17703393

👍

Reply:
Yeah, having ported every single use of six in cqlsh last year, I was pretty bewildered when it popped back up. Turns out it was just a bad merge.  Since the 3.25 python driver uses six, its mostly harmless to include it here, I guess.